packages/ember-glimmer/tests/integration/helpers/closure-action-test.js

```diff
@@ -2,7 +2,8 @@ import run from 'ember-metal/run_loop';
 import { computed } from 'ember-metal/computed';
 import isEnabled from 'ember-metal/features';
 import { subscribe, unsubscribe } from 'ember-metal/instrumentation';
-import { INVOKE } from 'ember-htmlbars/keywords/closure-action';
+import { INVOKE as HTMLBARS_INVOKE } from 'ember-htmlbars/keywords/closure-action';
+import { INVOKE as GLIMMER_INVOKE } from 'ember-glimmer/utils/references';
```

Comment:
Would be nice to put this forking into one of the helper files so that the test can just `import { INVOKE } from '../../utils/helpers';` and it would get either the HTMLBars or Glimmer symbol.